core/src/test/java/nl/nn/adapterframework/xml/XmlWriterTest.java

```diff
@@ -71,4 +71,20 @@ public void testNoLexicalHandling() throws Exception {
 		assertEquals(expected,xmlWriter.toString());
 	}
 
+	@Test
+	public void testDisableOutputEscaping() throws Exception {
```

Comment:
I think it would be a good idea not to introduce two new resources, but to add the ``&#xa;`` value to 'in.xml', to a element, to an attribute, to the escaped data in ``<data1>`` and to the CDATA in ``<data2>`` .
You'll also have to change quite some other resource files, but it makes the test cover more.